src/mathematics/variants.rs

```diff
@@ -0,0 +1,94 @@
+use truetype::GlyphID;
+use layout::Coverage;
+use super::Quantity;
+use std::io::SeekFrom;
+
+table! {
+    @position
+    #[doc = "A table of varaint glyphs to satisfy alternate measurements requirements."]
+    pub Variants { // MathVariants
+        min_connector_overlap      (u16),
+        vertical_coverage_offset   (u16), // VertGlyphCoverage
+        horizontal_coverage_offset (u16), // HorizGlyphCoverage
+        vertical_count             (u16), // VertGlyphCount
+        horizontal_count           (u16), // HorizGlyphCount
+
+        vertical_constructions (Vec<Construction>) |this, tape, position| {
+            let mut values: Vec<Construction> = Vec::with_capacity(this.vertical_count as usize);
+            let offsets: Vec<u16> = try!(tape.take_given(this.vertical_count as usize));
+            for n in offsets {
+                values.push(jump_take!(@unwrap tape, position, n as usize));
+            }
+            try!(tape.seek(SeekFrom::Start(position + 10 + 2*this.vertical_count as u64)));
```

Comment:
This is certainly not ideal -- I should probably just make these `Vec<u16>` and add `position` to each `offset` so that it's possible to seek to the required position.